src/natcap/invest/carbon.py

```diff
@@ -94,16 +94,16 @@
                 "A table that maps the land-cover IDs to carbon pools.  "
                 "The table must contain columns of 'LULC', 'C_above', "
                 "'C_Below', 'C_Soil', 'C_Dead' as described in the User's "

```

Comment:
Are these field names case sensitive? Either way, editing for some consistency with UG
```suggestion
                "'C_below', 'C_soil', 'C_dead' as described in the User's "

```

Reply:
I agree @emlys that case-_insensitive_ would be easiest for users (of ALL the models), but I would still advocate for a consistent use of capitalization between UI, UG, and actual column names used in the sample data.